src/Apps/__stories__/Apps.story.tsx

```diff
@@ -8,15 +8,15 @@ import { routes as collectRoutes } from "../Collect/routes"
 storiesOf("Apps", module)
   .add("Artwork Page", () => {
     return (
-      <ClientRouter
+      <MockRouter
```

Comment:
👍 I guess if we need a quick client-side router its easy enough to use `buildClientRouter` directly

Reply:
Probably yeah 👍 and likely best to do that once before we further abstract it